### PR TITLE
Link to IANA shc link

### PR DESCRIPTION
**Pull request by christianpaquin**
Added a ref to the SHC IANA registration: https://www.iana.org/assignments/uri-schemes/prov/shc.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ Issuer ->> Holder: Holder receives Health Card
...
+The `type`, and `credentialSubject` properties are added to the `vc` claim of the JWT. The `type` values are defined in [Credential Types](https://smarthealth.cards/vocabulary/); the `https://smarthealth.cards#health-card` MUST be present; other ones SHOULD be present as appropriate. The `issuer` property is represented by the registered JWT `iss` claim and the `issuanceDate` property is represented by the registered JWT `nbf` ("not before") claim (encoded as the number of seconds from 1970-01-01T00:00:00Z UTC, as specified by [RFC 7519](https://tools.ietf.org/html/rfc7519)).  Hence, the overall JWS payload matches the following structure (before it is [minified and compressed](#health-cards-are-small)):
```
Comment (p2-apple) [inline docs/index.md:219]: We use `SHALL` in place of `MUST` in HL7 parlance.

Comment (jmandel) [reply docs/index.md:219]: Agreed; can we revert this line and just apply the IANA fix?

Comment (christianpaquin) [reply docs/index.md:219]: Yes! oups, must not have started from clean slate...

**Comment by christianpaquin** [comment]
Fixed in #149.

**Pull request closed without merging**